### ISSUE TITLE
Enable quit keys setting isn't used in many places

### ISSUE THREAD

**Issue by kirstymcnaught** [OptiKey/OptiKey#363]
In Management console, Visuals tab, there is a setting for "Enable quit keys". This doesn't change the visibility or behaviour of the quit key in the main menu, which is the one I use. Most keyboards with quit keys do not respect this setting; from a cursory glance at the source, only English/SimplifiedAlpha and English/SimplifiedConversationAlpha seem to use it. 

Presumably this setting should apply to all instances of the quit key? Or if that isn't the intention, the setting description should be more specific, like "Enable quit keys in conversation keyboards".

**Comment by zelmon64** [comment]
@kirstymcnaught you are correct that only the _Simplified_ keyboards currently use this setting. The idea was that it will eventually effect the behaviour of the other keyboards.
